tests/nodeos_high_transaction_test.py

```diff
@@ -237,22 +255,46 @@ def findTransInBlock(transId, transToBlock, node):
             time.sleep(delayTime)
 
         transferAmount = Node.currencyIntToStr(round + 1, CORE_SYMBOL)
+
         Print("Sending round %d, transfer: %s" % (round, transferAmount))
         for accountIndex in range(0, args.total_accounts):
             fromAccount = accounts[accountIndex]
             toAccountIndex = accountIndex + 1 if accountIndex + 1 < args.total_accounts else 0
             toAccount = accounts[toAccountIndex]
             node = nonProdNodes[accountIndex % nonProdNodeCount]
-            trans=node.transferFunds(fromAccount, toAccount, transferAmount, "transfer round %d" % (round), exitOnError=False, reportStatus=False, sign = True)
-            if trans is None:
-                # delay and see if transfer is accepted now
-                Utils.Print("Transfer rejected, delay 1 second and see if it is then accepted")
-                time.sleep(1)
-                trans=node.transferFunds(fromAccount, toAccount, transferAmount, "transfer round %d" % (round), exitOnError=False, reportStatus=False, sign = True)
+            trans = None
+            attempts = 0
+            max_attempts = 3 if not args.send_duplicates else 1  # for send_duplicates we are just constructing a transaction, so should never require a second attempt
+            # can try up to max_attempts times to send the transfer
+            while trans is None and attempts < max_attempts:
+                if attempts > 0:
+                    # delay and see if transfer is accepted now
+                    Utils.Print("Transfer rejected, delay 1 second and see if it is then accepted")
+                    time.sleep(1)
+                trans=node.transferFunds(fromAccount, toAccount, transferAmount, "transfer round %d" % (round), exitOnError=False, reportStatus=False, sign = True, dontSend = args.send_duplicates)
+                attempts += 1
+
+            if args.send_duplicates:
+                sendTrans = trans
+                trans = None
+                numAccepted = 0
+                attempts = 0
+                while trans is None and attempts < 2:
```

Comment:
Well I guessed at 3 to allow us to catch up if we somehow exceeded the net staking for each account, but for the duplicates I think we could waste a lot of time trying to send it 3 times when likely it is because the p2p transaction got there before the duplicate did.  But decided to make both of them 2 and use the one constant.